src/game_constants.h

```diff
@@ -44,6 +44,9 @@
 #define OMAPX 180
 #define OMAPY 180
 
+// Size of a square unit of terrain saved to a directory.
+#define SEG_SIZE 32
```

Comment:
Depending on the lowest compiler versions used, this could be constexpr instead.